src/Analyzers/CSharp/Tests/UseCollectionExpression/UseCollectionExpressionForEmptyTests.cs

```diff
@@ -667,7 +667,7 @@ class C
             {
                 void M()
                 {
-                    MyList<int> x = {|CS9187:[]|};
+                    MyList<int> x = {|CS7036:[]|};
```

Comment:
curious.  what changed here?

Reply:
Previously, the error reported was:
```
error CS9187: Could not find an accessible 'Create' method with the expected signature:
    a static method with a single parameter of type 'ReadOnlySpan<T>' and return type 'MyList<T>'.
```

Now, the error reported is:
```
error CS7036: There is no argument given that corresponds to the required parameter 'x' of
    'MyList.Create<T>(ReadOnlySpan<T>, int)'
```

Reply:
gotcha.  would it make sense to refine the message to say something like "Are you missing an `args(...)` element?"